packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <KPop
     :hide-popover="hidePopover"
+    v-bind="position"
```

Comment:
Why would we ever want this set to false?

instead of using `v-bind` just use the KPop prop. 

```suggestion
    :position-fixed="position"
```

Reply:
Ended up leaving this alone, we probably want to default to position fixed no matter what.